src/templates/partials/json-schema.swig

```diff
@@ -13,23 +13,22 @@
 	"url" : "{{site.url}}",
 	"logo": {
   		"@type": "ImageObject",
-  		"url": "{{site.url}}/images/logos/gwwc-logo-schema-126x60.png",
-  		"width": 126,
-  		"height": 60
+  		"url": "{{site.url}}/images/logos/gwwc-logo-schema-white-shadow-260x260.png",
```

Comment:
Random, may just be worth submitting a few pages from the build preview to the [Schema.org validator](https://validator.schema.org/), I recall that we sometimes got errors with this (usually from unescaped strings)